### DISCUSSION TITLE
Security vulnerability on jquery packages

### DISCUSSION THREAD

**Discussion by ZenwalkerD** [jquery/jquery#5043] [Q&A] [answered]
### Description ###
The latest package of Jquery has a security vulnerability appearing in the vulnerability reports on my project. 
Below i have pasted the vulnerability issue i have. 

### Link to test case ###
>jQuery contains commented references to the hijacked domain `blindsignals`, within the files `src/queue/delay.js` and >`test/data/jquery-1.9.1.js` (the former referring to a Web Archive version of the original site). Users without awareness of the >domain's status could be exposed to unspecified attacks if they attempt to follow the links to the hijacked site.

Any suggestions are helpful.

Thanks

**Comment by mgol** [comment, marked as answer]
jQuery 3.6.1 with the fix is out now: https://blog.jquery.com/2022/08/26/jquery-3-6-1-maintenance-release/
